project/src/main/webapp/index.html

```diff
@@ -111,18 +112,22 @@ <h2>Food Mood's recommendation:</h2>
         </div>
         <div id="results-map-container"> </div>
       </div>
-      <div id="feedback-box">
+      <div id="feedback-box" style="display: none">
         <label for="liking-feedback">Like these options?<br> Let us know what you chose: <br></label>
-        <select class="button" name="feedback" id="feedback">
-          <option value="1">First option</option>
-          <option value="2">Second option</option>
-          <option value="3">Third option</option>
-          <br>
-          <input type="submit" value="Submit" style="font-weight: bold">
-        </select>
+        <form id="chosen-place-form" class="form-element-toolbar">
```

Comment:
(Not for this PR):
a nicer UX would be to actually populate the options with the restaurants' names. Shouldn't be too hard.

Reply:
Good idea, I added an issue